WordPress/theme-experiments
# [Parisienne] Pull in block starter content from dedicated directory

Following #16, this PR separates starter content in the Parisienne theme from block template parts — which should conceptually be reserved for aspects of the site that are used across pages, not demo content or page templates.

To test:

1. Create a fresh WordPress site
2. Check out this branch and clone it into your site's themes directory
3. Install the Gutenberg plugin
4. Enable the "Full Site Editing" experiment in the Gutenberg > Experiments panel.
5. Activate `Parisienne` from within Appearance > Themes.
6. Go to Customizer and hit Publish
7. Visit the site and ensure that the `Home`, `About`, and `Contact` pages have loaded with content in each of them.
